config.py

```diff
@@ -17,6 +17,8 @@ class Config:
     RABBITMQ_SURVEY_LAUNCHED_ROUTING_KEY = os.getenv('RABBITMQ_SURVEY_LAUNCHED_ROUTING_KEY',
                                                      'events.caseProcessor.surveyLaunched')
     RABBITMQ_DEACTIVATE_UAC_QUEUE = os.getenv('RABBITMQ_DEACTIVATE_UAC_QUEUE', 'events.caseProcessor.deactivateUac')
+    RABBITMQ_UPDATE_SAMPLE_SENSITIVE_QUEUE = os.getenv('RABBITMQ_UPDATE_SAMPLE_SENSITIVE_QUEUE',
```

Comment:
Should be `RABBITMQ_UPDATE_SAMPLE_SENSITIVE_ROUTING_KEY` (the fact it's also the queue name is a mistaken pattern we'll be getting rid of).